BGAnimations/ScreenEvaluation common/default.lua

```diff
@@ -1,5 +1,5 @@
 local Players = GAMESTATE:GetHumanPlayers()
-local NumPanes = SL.Global.GameMode=="Casual" and 1 or 7
+local NumPanes = SL.Global.GameMode=="Casual" and 1 or 8

```

Comment:
Isn't it only 7 for FA+ mode?

Reply:
The way panes work is that the value is the total number of panes available for the game.

If panes are unused in specific cases, the default.lua for the pane just returns nil so they don't actually get included in the list to show. This way the naming of the panes is always still consistent.

This is handled in the default.lua of pane8
```lua
if SL.Global.GameMode ~= "ITG" then
	return
end
```

I will say there is some room for cleanup on how panes are handled in general, but that's for another day.